internal/factsengine/gatherers/corosyncconf.go

```diff
@@ -83,26 +82,10 @@ func (s *CorosyncConfGatherer) Gather(
 	}
 
 	log.Infof("Requested corosync.conf file facts gathered")
-	return facts, nil
-}
-
-func readCorosyncConfFileByLines(filePath string) ([]string, error) {
-	corosyncConfFile, err := os.Open(filePath)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not open corosync.conf file")
-	}
-
-	defer corosyncConfFile.Close()
-
-	fileScanner := bufio.NewScanner(corosyncConfFile)
-	fileScanner.Split(bufio.ScanLines)
-	var fileLines []string
-
-	for fileScanner.Scan() {
-		fileLines = append(fileLines, fileScanner.Text())
+	if ctx.Err() != nil {
```

Comment:
Do we need to check this?
I mean, `utils.ScanFileContext`is already returning an error in this case.
I wouldn't worry to check this thinking that between the scan file and this small piece of code we could have the cancellation.
If we are going to do so in all the gatherers (I think it is an overkill) I think we should do it in the `gathering.go` code.
There we already have a `switch` after running the `Gather`, so we could check there.
The purpose of thie context is to not block rather than notifying an error